src/main/java/de/tum/in/www1/artemis/repository/tutorialgroups/TutorialGroupRegistrationRepository.java

```diff
@@ -19,6 +22,16 @@ Optional<TutorialGroupRegistration> findTutorialGroupRegistrationByTutorialGroup
 
     Set<TutorialGroupRegistration> findAllByTutorialGroupAndType(TutorialGroup tutorialGroup, TutorialGroupRegistrationType type);
 
+    @Modifying
+    @Transactional
     void deleteAllByStudent(User student);
 
+    @Modifying
+    @Transactional
+    void deleteAllByStudentIsInAndTypeAndTutorialGroup_Course(Set<User> students, TutorialGroupRegistrationType type, Course course);
```

Comment:
We're not using underscores for normal methods in our code base

Reply:
Removed the underscores in 8425cbdf3c0fe6a84b3188d9cab98e20a3de3c5d